lib/ex_spirit/parser/text.ex

```diff
@@ -530,13 +531,13 @@ defmodule ExSpirit.Parser.Text do
                 }
               parser when is_function(parser, 1) ->
                 %{context |
-                  position: context.position + byte_size(<<c::utf8>>),
+                  position: context.position,
```

Comment:
As this line is a no-op now other than just causing an extra set (needless cost), can you just delete it entirely?  :-)